generator/source/faq/consensus.rst

```diff
@@ -77,8 +76,7 @@ But you can change consensus after the Sawtooth network has started.
 
 Is PoET CFT suitable for production use?
 ----------------------------------------
-
-Yes.  It is for systems that do not have SGX and is intended for use in production.  Both PoET CFT and PoET SGX have tests to guard against bad actors, such as the "Z Test" to check a validator is not winning too frequently.
+Yes. It is for systems that do not have SGX and is intended for use in production for systems that do not require BFT. Both PoET CFT and PoET SGX have tests to guard against bad actors, such as the "Z Test" to check a validator is not winning too frequently.
```

Comment:
Accepted. I'll remove mention of production-readiness. Here is the reworded answer:

> What protections does PoET CFT have, since it is not BFT?
> ----------------------------------------
> It is for systems that do not have SGX and do not require BFT. Both PoET CFT and PoET SGX 
have tests to guard against bad actors, such as the "Z Test" to check a validator is not winning too frequently.  PoET CFT simulates the SGX environment and provides CFT. That said, PoET SGX is preferred because of the additional SGX protections for generating the wait time.